processor/tests/test_staging.py

```diff
@@ -260,3 +261,60 @@ def get_id_set(path, part):
             set_b = get_id_set(path_b, part)
             assert len(set_a) > 0
             assert set_a == set_b
+
+
+def test_staging_run_incremental(moz_fx_data_stage_data, tmpdir):
+    output = Path(tmpdir / "output")
+    runner = CliRunner()
+    runner.invoke(
+        staging.run,
+        [
+            "--date",
+            BASE_DATE,
+            "--input",
+            f"{moz_fx_data_stage_data}",
```

Comment:
```suggestion
            moz_fx_data_stage_data,
```

Reply:
The path needs to be stringified, so either leaving it as is or wrapped in a `str` call.

Reply:
assumed it was already a string. Either works in the case, perhaps just casting it to string is a little better (less ambiguous what's happening)